images/br-federation-manager/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM alpine:3.14
+FROM pingcap/tools-base:v1
```

Comment:
what's the difference between `pingcap-base` and `tools-base`?

Reply:
tools-base contains bind-tools, and wget, which some tools usually need

Reply:
change to single pingcap-base